### ISSUE TITLE
Write single javascript function implementing Activity Sequence from UML

### ISSUE THREAD

**Issue by aenkya** [Enkya/ticket-booking-vanilla#3]
We need an initial simple _function_ to simulate the performance of the booking algorithm, written as one function in **JavaScript**

**Comment by shyakaster** [comment]
Do we need to have UML diagram in place before we attempt to do this?

**Comment by aenkya** [comment]
Yes we do. If you are done generating one, you can share a screenshot of it here
